runelite-client/src/main/java/net/runelite/client/util/OSType.java

```diff
@@ -36,13 +36,7 @@
 
 	private static OSType cached;
```

Comment:
make this variable constant (make it final, and use UPPER_CASE) and rename it to something that makes more sense without context, so OS_TYPE would be fine here.